### DISCUSSION TITLE
abilitare "mostra dettagli di pagamento"

### DISCUSSION THREAD

**Discussion by spokke** [fattureincloud/api#247] [Q&A] [answered]
salve,
è possibile tramite richiesta API create issued document abilitare il checkbox visualizza i dettagli di pagamento nella fattura generata ?
se si qual è il campo da passare ? 
grazie

**Comment by emajo** [comment, marked as answer]
Ciao @spokke, il campo in questione è `show_payment_method`, ovviamente deve essere valorizzato il campo `payment_method.id` con l'id di un metodo di pagamento esistente.

Grazie,
Emanuele.

**Reply by spokke** [reply]
ok grazie funziona !
